.github/workflows/ios-ci.yml

```diff
@@ -332,13 +332,30 @@ jobs:
         run: |
           VERSION=${{ env.version }} COCOAPODS_TRUNK_TOKEN=${{ secrets.COCOAPODS_PASSWORD }} pod trunk push MapLibre.podspec
 
+  ios-build-cmake:
+    needs: pre_job
+    runs-on: macos-14
```

Comment:
Should we matrix this to run across all our supported major version? Do we support 13, 14, and 15?

Reply:
No we only support building with a recent version of Xcode. I guess we could use `macos-latest` instead here.